### ISSUE TITLE
Running `make run` fails when assets are involved

### ISSUE THREAD

**Issue by jwoertink** [jwoertink/fez#24]
Need to make sure this exists.

```
ruby es2js.rb
Traceback (most recent call last):
	1: from es2js.rb:13:in `<main>'
es2js.rb:13:in `write': No such file or directory @ rb_sysopen - public/javascripts/application.js (Errno::ENOENT)
make: *** [assets] Error 1
```

**Comment by Kanezoh** [comment]
Hello, I've just ran into this bug.
As a solution, just adding statement to make empty application.js isn't enough?
or application.js needs any code?

**Comment by jwoertink** [comment]
I think just having an empty application.js file may be enough. I haven't looked in to it much, but I think that's all that's needed.

**Comment by Kanezoh** [comment]
#25 How about this? It's too simple, but works fine.

**Comment by jwoertink** [comment]
Fixed by #25
